Sources/FanMenu.swift

```diff
@@ -198,7 +198,8 @@ class FanMenuScene {
         
         node = [backgroundCircle, buttonsNode, buttonNode].group()
         
-        buttonNode.onTouchPressed { _ in
+        buttonNode.onTouchPressed { [weak self] _ in
```

Comment:
I think here we can use unowned here because node without View - it shouldn't be possible in this case